nvm.sh

```diff
@@ -335,21 +335,40 @@ nvm_find_nvmrc() {
   fi
 }
 
-# Obtain nvm version from rc file
+# Obtain the nvm version from the .nvmrc file
 nvm_rc_version() {
+  # Check if quiet mode is on
+  local quiet="false"
+  if [ "$1" == "--silent" ]; then
+    quiet="true"
+  fi
+  # Set NVM_RC_VERSION to null before we set it ourselves
```

Comment:
we don't need all these comments; a comment that explains what the next line is doing verbatim isn't helping the code be readable :-)
```suggestion
```